modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -43,7 +44,7 @@ You can only keep RAML API specifications as local copies within your project's
 
 You can start xref:create-api-specification-studio.adoc[an API specification project in Studio and synchronize it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
 
-You can also import an API specification from Design Center as a an API specification project with VCS enabled capabilities to edit your API specification using Studio and synchronize your changes with Design Center:
+You can also import an API specification from Design Center as an API specification project with VCS enabled capabilities to edit your API specification using Studio and synchronize your changes with Design Center:
```

Comment:
```suggestion
You can also import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and synchronize your changes with Design Center:
```

Reply:
I am confused by this paragraph. Is it the VCS-enabled capabilities that edit the API specification? It might be clearer if you break this up.

Reply:
OK! Thanks!